src/aggregate.h

```diff
@@ -119,7 +119,7 @@ class AggregateDeclaration : public ScopeDsymbol
     Expression *getRTInfo;      // pointer to GC info generated by object.RTInfo(this)
 
     AggregateDeclaration(Loc loc, Identifier *id);
-    void setScope(Scope *sc);
```

Comment:
It has removed in #5500 (https://github.com/dlang/dmd/commit/75b5b691555bef2d2f21b44821f297f93ddfbc77#diff-d9f55fb03991f402fab0f1fdec9effb2L116), but I had forgotten updating C++ header.